app/services/albums.js

```diff
@@ -0,0 +1,9 @@
+const axios = require('axios');
+const errors = require('../errors');
+const config = require('../../config');
```

Comment:
can you use destructuring to get the albumsBaseUrl?
is a little bit unprettier to have lines with config.common.api.albumsBaseUrl, is too large